### ISSUE TITLE
Some questions about the reproduction and testing process

### ISSUE THREAD

**Issue by Warrior456** [donydchen/mvsplat#61]
Hi, thanks for sharing this amazing work.

I used the pre-trained model provided for evaluation.

For re10k, the metric is lower than that in the paper, and I don't know why it will end early (end the test at `6474/7286`).

![image](https://github.com/user-attachments/assets/6e9fc46c-4b83-4bfa-a95d-60eb369f4ff4)

For ACID, I got metrics similar to those in the paper, but the testing process also ended early.
![image](https://github.com/user-attachments/assets/41e36b60-dda5-49f9-9574-355910bd199c)

Looking forward to your reply

**Comment by donydchen** [comment]
Hi @Warrior456, thanks for your interest in our work.

I just ran a test using the newest released code on my machine, and the scores matched precisely with our paper reported ones. See below

![re10k results](https://github.com/user-attachments/assets/8abc4fc4-5b68-412a-b351-009717bd099c)

I reckoned some minor differences among different versions of some Python packages might cause this. Hence, I added a [requirements_w_version.txt](https://github.com/donydchen/mvsplat/blob/main/requirements_w_version.txt) for your reference. I would appreciate your reply if matching the packages' version helps get the correct scores.

Besides, regarding why the test ended earlier at `6474/7286`, this is as expected and nothing to be worried about. In short, this is because some scenes are skipped, more details can be found at https://github.com/donydchen/mvsplat/issues/60#issuecomment-2311703128.

**Comment by donydchen** [comment]
Hi @Warrior456, glad to know that my previous suggestion helped.

* I feel MVSNeRF (Anpei Chen et al. ICCV21) might struggle with wide-baseline data since it relies on one single cost volume built upon one randomly selected source view. In this case, the quality of the rendered novel views depends on how the source and reference views are selected and how far away the target views differ from the input ones, which are both challenging regarding wide-baseline inputs. On the other hand, MatchNeRF might work better as it relies on feature matching, which essentially resembles a camera frustum volume built on the *target view* (not the source view). But still, in the extreme case when the wide baseline inputs contain very limited overlap, MatchNeRF will fail to find any meaningful matching information, leading to unsatisfactory outputs. Overall, both MVSNeRF and MatchNeRF are reasonable comparison methods, and you can try them to see how the results go.

* Correct. We set the depth range as 1~100 for both RE10K and ACID, and more discussions can be found at https://github.com/donydchen/mvsplat/issues/11#issuecomment-2026812577

* With the default settings (batch size: 14, training step: 300K), training our MVSplat takes around 5.5 days on a single 80G A100 GPU.

**Comment by Rashfu** [comment]
Hi @donydchen , I have a question regarding monitoring the training process. I would like to track the training progress, similar to how it's done in TensorBoard. However, during my training, I noticed that the files in the output directory do not seem to contain training information. In particular, the *.log files are empty. Is there a specific configuration I need to enable? 

Thanks in advance !
